src/internal/testing/TestScheduler.ts

```diff
@@ -409,14 +413,62 @@ export class TestScheduler extends VirtualTimeScheduler {
     this.maxFrames = Infinity;
     this.runMode = true;
     AsyncScheduler.delegate = this;
+    dateTimestampProvider.delegate = this;
+    performanceTimestampProvider.delegate = this;
 
-    const helpers = {
+    let animationFramesHandle = 0;
+    let animationFramesQueue: Map<number, FrameRequestCallback> | undefined;
+
+    requestAnimationFrameProvider.delegate = {
+      requestAnimationFrame(callback) {
+        if (!animationFramesQueue) {
+          throw new Error("repaints() was not called within run()");
+        }
+        const handle = ++animationFramesHandle;
+        animationFramesQueue.set(handle, callback);
+        return handle;
+      },
+      cancelAnimationFrame(handle) {
+        if (!animationFramesQueue) {
+          throw new Error("repaints() was not called within run()");
+        }
+        animationFramesQueue.delete(handle);
+      }
+    };
+    const repaints = (marbles: string) => {
```

Comment:
I can easily have my mind changed on this... but I think we should probably move this out to have it in the same place as all the other helpers. I know that makes it available for "non-run-mode" use, and maybe we want to avoid that.. but I do like consistency.

Reply:
Oh... on more thing, I'm not sure I like `repaints`, as it's not readily obvious that it is associated to "animation frames". Perhaps a different name here would be better?  `animates`? `animationFrames`? (the latter might be too confusing though).

Reply:
> Perhaps a different name here would be better? 

How about `animate`? It doesn't return anything, so I suppose it doesn't need the plural noun that would usually be used for an observable.